### DISCUSSION TITLE
Can't make Upload File work. In need of an example.

### DISCUSSION THREAD

**Discussion by eduard-buta** [AutomaApp/automa#294] [Q&A] [answered]
Hi. I can't seem to make Upload File work.

I've made sure to allow the extension to use my files, and I've also tried following the Google Translate example put out in the Marketplace. I couldn't make that work either. 

Maybe it's just me not understanding how to properly input the selector and the path. Could someone please help me out a bit?

Edit: This is the website I'm trying to figure the input field Selector from. https://cloudconvert.com/txt-to-doc

**Comment by Kholid060** [comment, marked as answer]
To see how to input the selector and the file path on the upload file block, you can open [Automa docs](https://docs.automa.site/blocks/upload-file.html). And for the selector try `input[type="file"]`.

To upload a file to cloudconvert.com try below workflow (replace `.txt` to `.json` and import it)
[cloudconvert.txt](https://github.com/Kholid060/automa/files/7963059/cloudconvert.txt)

And if the workflow not working, try to check its logs to see if there's an error or not.

**Reply by techflag** [reply]
there are two or more ,how to choose the right input[type="file"] to upload file

**Reply by Kholid060** [reply]
I guess try the input[type="file"] one by one to see which one is the input for uploading a file.

**Reply by techflag** [reply]
multiple  input[type="file"] ，like this
![image](https://user-images.githubusercontent.com/939362/153740655-75193d8b-8e88-4781-aaf8-8e05af6f0f67.png)

**Reply by luo57wuqi** [reply]
I'm using file upload. If the file is larger than 48, it failed..message:failed to fetch
